Scheduler/tests/query_and_cancel_test.py

```diff
@@ -0,0 +1,177 @@
+import json
+import logging
+import sys
+
+from datetime import datetime
+
+import boto3
+
+from job_requester import JobRequester
+from job_requester import Message
+
+
+"""
+How tests are executed:
+- Put tickets on the request queue, in-progress pool and dead letter queue; create Message objects (request identifiers)
+that correspond to these tickets.
+- Create a JobRequester object and call query_status on the identifiers, check that the statuses returned are corrected.
+- Call cancel_request on tickets on the request queue, check that the request tickets are removed. 
+- Clean up the artifacts. 
+"""
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+LOGGER.addHandler(logging.StreamHandler(sys.stdout))
+
+# test parameters
+TEST_ECR_URI = "754106851545.dkr.ecr.us-west-2.amazonaws.com/pr-tensorflow-training:2.2.0-gpu-py37-cu101-ubuntu18.04-example-pr-269-2020-06-11-22-13-27"
```

Comment:
Change this to use an image from one of the already released images:

https://github.com/aws/deep-learning-containers/blob/master/available_images.md#general-framework-containers

Use the TensorFlow 2.2.0 training image in the table above:
```
763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-gpu-py37-cu101-ubuntu18.04
```